spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_4_async_spec.rb

```diff
@@ -1,15 +1,14 @@
 describe ManageIQ::Providers::Redhat::InfraManager::Refresh::Refresher do
   before(:each) do
     _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
-    @ems = FactoryGirl.build(:ems_redhat_with_ensure_managers, :zone => zone, :hostname => "localhost", :ipaddress => "localhost",
+    @ems = FactoryGirl.create(:ems_redhat, :zone => zone, :hostname => "localhost", :ipaddress => "localhost",
```

Comment:
same question

Reply:
Previously I changed it to 'build' since I had to mock the 'ovirt_service' result before the 'save', this is not needed any more, so I reverted it back to create.